client/src/planwise/client/ui/common.cljs

```diff
@@ -35,10 +35,10 @@
    [header {:sections sections :account account :title title :tabs tabs :action action :secondary-actions secondary-actions}]])
 
 (defn full-screen
-  [{:keys [sections account title tabs action footer main-prop main secondary-actions]} & children]
+  [{:keys [sections account title tabs action footer main-prop main secondary-actions sidebar-prop]} & children]
   [:div.layout.full-screen
    [:main main-prop main]
-   (into [:aside {:id "sidebar"}] children)
+   (into [:aside (merge {:id "sidebar"} sidebar-prop)] children)
```

Comment:
Minor idiomatic suggestion: you can use `#sidebar` to assign IDs to DOM nodes. Ie. this can be written as:
```clojure
(into [:aside#sidebar sidebar-prop] children)
```